components/src/core/components/CardTable/CardTable.vue

```diff
@@ -85,6 +85,14 @@ export default defineComponent({
       type: String,
       default: () => nanoid(8),
     },
+    styleClass: {
```

Comment:
why not use [class attribute](https://v3.vuejs.org/guide/component-attrs.html#non-prop-attributes) directly?

Reply:
used the class attribute directly